packages/hooks/use-namespace/index.ts

```diff
@@ -70,9 +70,9 @@ export const useNamespace = (
       ? _bem(namespace.value, block, blockSuffix, element, modifier)
       : ''
   const is: {
-    (name: string, state: boolean | undefined): string
+    (name: string, state: boolean | string | undefined): string
     (name: string): string
-  } = (name: string, ...args: [boolean | undefined] | []) => {
+  } = (name: string, ...args: [boolean | string | undefined] | []) => {
```

Comment:
The `state` support string looks quite strange, perhaps we can avoid modifying it here